### PR TITLE
Lesson 02 task

### PR DESCRIPTION
**Pull request by karamel29**

### CODE REVIEW

File: task002/main.go [hunk 1]
```diff
@@ -0,0 +1,13 @@
...
+import (
+	"fmt"
+
+	"github.com/karamel29/go-courses/task002/fibonacci"
+)
+
+func main() {
+	fmt.Println("Hi")
+	defer fmt.Println("Mission accomplished")
+	fibonacci.Printer(5)
```
Comment (DzmitryYafremenka) [inline task002/main.go:12]: Although the original Fibonacci sequence did start with 1 and 2 nowadays it is no longer considered correct. Now it is: 0, 1, 1, 1, 2, 3, 5, 8, 13, 21, 34, 55, 89, 144, 233, 377, 610, 987, 1597. How do you change the function to get the initial values (0, 1)?

Comment (karamel29) [reply task002/main.go:12]: @DzmitryYafremenka **To get the initial values (0, 1):**
```
func fibonacci() func() int {
 i, j := 0, 1
 return func() int {
  nextFibonacci := i + j
  result := i
  i, j = j, nextFibonacci
  return result
 }
}
```

**Review by DzmitryYafremenka — APPROVED** [review]

File: task002/fibonacci/fibonacci.go [hunk 2]
```diff
@@ -0,0 +1,19 @@
+package fibonacci
+
+import "fmt"
+
+func fibonacci() func() int {
+	i, j := 0, 1
+	return func() int {
+		nextFibonacci := i + j
```
Comment (IgorZhadenovEpam) [inline task002/fibonacci/fibonacci.go:8]: It is common in Go to use short variable names. Variable name 'next' would be more readable then 'nextFibonacci'.

**Review by IgorZhadenovEpam — APPROVED** [review]
Overall good!

**Pull request merged**